stacks_queues/stack.py

```diff
@@ -1,18 +1,20 @@
-from stacks_queues.linked_list import LinkedList
+# from stacks_queues.linked_list import LinkedList
 
 class StackEmptyException(Exception):
     pass
 
 class Stack:
```

Comment:
The assignment was to use the `LinkedList` as the internal data structure for the Stack.